search-app/src/cmr/search/services/query_execution/facets/hierarchical_v2_facets.clj

```diff
@@ -409,14 +447,24 @@
   that field."
   [field bucket-map base-url query-params]
   (let [field-hierarchy (get-field-hierarchy field query-params)
-        hierarchical-facet (-> (parse-hierarchical-bucket-v2 field field-hierarchy base-url
-                                                             query-params bucket-map)
+        hierarchical-facet (-> (parse-hierarchical-bucket-v2
+                                 field
+                                 field-hierarchy
+                                 base-url
+                                 query-params
+                                 bucket-map)
```

Comment:
This makes it look kinda like you're threading the params of `parse-hierarchical-bucket-v2` rather than what is actually happening. Maybe just add a binding for the results of that function and thread it?